model/PluginRegister.php

```diff
@@ -130,21 +130,44 @@ public function getTemplates($names=null) {
     }
 
     /**
-     * Returns an array of javascript function names to call when loading pages
+     * Returns an array of plugin callback function names
+     * @param array $names the plugin name strings (foldernames) in an array
+     * @return array
+     */
+    public function getPluginCallbacks($names=null) {
+        if ($names) {
```

Comment:
The tests don't cover the case of $names being passed. I think it should be easy to modify the test for getPluginCallbacks. I will suggest below...